site/src/docs/getting-started/designer.mdx

```diff
@@ -30,58 +30,42 @@ Always keep these core principles in mind when making design decisions:
 
 Helsinki Grotesk font can be purchased from <ExternalLink href="https://camelot-typefaces.com/helsinki-grotesk">Camelot Typefaces website</ExternalLink>
 
-### Set up your Abstract project
-To set up a project in Abstract, you will need to:
+### Set up your Figma project
+To set up a project in Figma, you will need to:
 
-1. Create a new Abstract project
-2. Add at least one design (Sketch) file to the project
-3. Link HDS libraries to your project
-4. Create a branch and start making design changes
+1. Create a new Figma project in All Helsinki projects-team
+2. Add a new design file
+3. Add HDS Design Kit library to your project
+4. Start making design changes
 
-**If you are new to Abstract**, please read <InternalLink href="/getting-started/tutorials/abstract-tutorial">HDS Getting started with Abstract tutorial</InternalLink> for more information on how to set up and use Abstract.
 
 ## Get familiar with HDS
 Before you start designing, it is a good idea to get familiar with the HDS offering.
 
-For designers, a good place to start is to browse through design collections in the HDS Abstract project. 
-- <ExternalLink href="https://share.goabstract.com/7b92ffce-5566-4aa1-be7a-3555b8d7a11c">Abstract Collection - Components</ExternalLink>
-- <ExternalLink href="https://share.goabstract.com/1dd17c91-0b9e-463a-9492-66af1301e52a">Abstract Collection - Design Tokens</ExternalLink>
-- <ExternalLink href="https://share.goabstract.com/cd813835-b6de-42a7-8686-0ea8aec21663">Abstract Collection - Patterns</ExternalLink>
-- <ExternalLink href="https://share.goabstract.com/1dd17c91-0b9e-463a-9492-66af1301e52a">Abstract Collection - Design Tokens</ExternalLink>
-- <ExternalLink href="https://share.goabstract.com/34245a50-f1a5-4160-9059-c4ccf320c7d2">Abstract Collection - Icons</ExternalLink>
-- <ExternalLink href="https://share.goabstract.com/e6aa2ebd-1a49-4cb1-90ef-a44a6486c28b">Abstract Collection - Example Custom Components</ExternalLink>
-
-### Customising component symbols
-The component symbols use _smart layout_ for easy resizing. You can also configure the content and styling of symbol parts from the _Overrides_ section of symbol properties.
+For designers, a good place to start is to browse through <ExternalLink href="https://www.figma.com/file/Kxwg3R0zNRHj55nQqFu6VS/HDS-UI-Kit-v3?type=design&node-id=2688%3A137088&mode=design&t=JMnhoneDpxxvmQfg-1">HDS Design Kit overview</ExternalLink>
 
-_When changing the colour or other styles of component symbols, be sure your custom design adheres to the design guidelines._ Styles within components are carefully considered. We do not recommend detaching the symbol to change default styles.
 
-<Image src="/images/getting-started/designers/component-overrides.png" alt="Customising symbol overrides" style={{ width: '100%', maxwidth: 850, marginbottom: 24 }} viewable/>
+### Customising components
```

Comment:
Not changed in this PR, but level of this heading and next one ("Styling text elements") seems to be too low? 

Are they meant to be under "Get familiar with HDS"?

Reply:
Yes, I suppose, as they refer directly to stuff you can do with the HDS design offering